OHHTTPStubs/UnitTests/Test Suites/MocktailTests.m

```diff
@@ -45,14 +45,15 @@ - (void)setUp
 {
     [super setUp];
     [OHHTTPStubs removeAllStubs];
-    
+
```

Comment:
Would it be possible to get rid of those changes? 🙏 (by using the trick in the Xcode prefs I mentioned you in the last review or even manually of you don't want to change your prefs, that works too)

Basically all empty lines should really be empty and not contain indentation and spaces. I know it's nitpicking and a matter of taste, I don't care for one style or another but I care about consistency throughout the code base 😉